support-workers/src/main/scala/com/gu/support/workers/lambdas/ServicesHandler.scala

```diff
@@ -9,7 +9,7 @@ import io.circe.{Decoder, Encoder}
 import scala.concurrent.duration._
 import scala.concurrent.{ExecutionContext, Future}
 
-abstract class ServicesHandler[T <: StepFunctionUserState, R](servicesProvider: ServiceProvider = ServiceProvider, d: Option[Duration] = None)(
+abstract class ServicesHandler[T <: StepFunctionUserState, R](servicesProvider: ServiceProvider, d: Option[Duration] = None)(
```

Comment:
I have removed the default value for the `servicesProvider` constructor parameter to force overriding classes to pass this on and hopefully prevent the same bug happening again.

Reply:
totally separate comment to this PR, but I'm not a massive fan of inheritance and overriding as dependencies are less explicit and harder to not have.  However I do see that it makes a stuff available "for free" without as much boilerplate.  It's the old "is-a vs has-a" debate or "composition vs inheritance".
In this case it's clear that there's very little in the `ServicesHandler` so things are not bad at the moment, as long as it doesn't expand.